android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -788,27 +652,6 @@ protected static int getKeyboardIndex(Context context, String keyboardKey) {
     return index;
   }
 
-  /**
-   * Get the list of associated keyboards for a given language ID
-   * @param langId
-   * @return ArrayList of keyboard
-   */
-  public static ArrayList<HashMap<String, String>> getAssociatedKeyboards(String langId) {
```

Comment:
It seems this function has been eliminated?  Was nothing referencing it?

Reply:
Currently, there's no references to it.
I had added this during the original implementation of [LanguagesSettingsActivity.java](https://github.com/keymanapp/keyman/pull/1760/files#diff-06e214859ead084638ac3396c8e15e3b) l. 76  to get a list of keyboards for a given language ID.

That was replaced when you refactored to use Dataset instead.